### PR TITLE
move some python CI checks to github actions

### PR DESCRIPTION
**Pull request by jgrund**
Signed-off-by: Joe Grund <jgrund@whamcloud.io>

---
This change is [Reviewable](https://reviewable.io/reviews/whamcloud/integrated-manager-for-lustre/2369)

### CODE REVIEW

**Review by ip1981 — APPROVED** [review]

File: tests/unit/chroma_core/helpers/helper.py [hunk 1]
```diff
@@ -61,11 +61,13 @@ def generate_csr(common_name):
         stdout=subprocess.PIPE,
         stderr=subprocess.PIPE,
     )
+
```
Comment (johnsonw) [inline tests/unit/chroma_core/helpers/helper.py:64]: nit: are these empty lines necessary? Might be able to just revert this file.

**Review by johnsonw — APPROVED** [review]

File: tests/unit/chroma_core/helpers/helper.py (hunk 1 above)
Comment (johnsonw) [reply tests/unit/chroma_core/helpers/helper.py:64]: Or maybe `black` updated the fie?

Comment (jgrund) [reply tests/unit/chroma_core/helpers/helper.py:64]: I added them. I agree they don't serve any real purpose besides statement separation, but don't see any harm in them being here.

Comment (johnsonw) [reply tests/unit/chroma_core/helpers/helper.py:64]: for sure 👍

**Pull request merged**